h/activity/views.py

```diff
@@ -54,13 +54,11 @@ def eager_load_documents(query):
         timeframes = bucketing.bucket(anns)
 
         for timeframe in timeframes:
-            for document, annotations in timeframe.document_buckets.items():
-                results = []
-                for annotation in annotations:
+            for document, bucket in timeframe.document_buckets.items():
+                for index, annotation in enumerate(bucket.annotations):
```

Comment:
Yup, of course. I think as soon as the frontend for this page is underway we should look at the views code, and also add tests (there's only one at the moment checking for the feature flag).